constraints/miden-vm/bitwise.air

```diff
@@ -0,0 +1,73 @@
+def BitwiseAir
+
+# Enforces that column must be binary
+ev check_binary(main: [a]):
+    enf a^2 - a = 0
+
+# Returns value aggregated from limbs
+fn aggregate(limb: vector[4]) -> scalar:
+    let result = sum([2^i * a for (i, a) in (0..4, limb)])
+    result
```

Comment:
nit: Return statements would require `return` keyword. And maybe we can remove the result variable as well.